tests/core/test_expectation_suite.py

```diff
@@ -185,6 +193,137 @@ def __dict__(self):
         )
 
 
+class TestCRUDMethods:
+    """Tests related to the 1.0 CRUD API."""
+
+    expectation_suite_name = "test-suite"
+    store_key = ExpectationSuiteIdentifier(
+        expectation_suite_name=expectation_suite_name
+    )
+
+    @pytest.fixture
+    def expectation_params(self):
+        return {"column": "a", "value_set": [1, 2, 3], "result_format": "BASIC"}
```

Comment:
why not make the fixture the expectation itself? If we change the signature of `ExpectColumnValuesToBeInSet`, it'd be nice for the type system to catch any missing values before we run the test.

Reply:
great suggestion, updated in [e064aa7](https://github.com/great-expectations/great_expectations/pull/8986/commits/e064aa749341744d87b48c02cef3f873cc157c53)